### PR TITLE
CDM-27: Create Stepper component

### PR DESCRIPTION
**Pull request by danielaprado**
### Description

- Create Stepper component
- Desktop & Mobile responsive

### Attachments
- **Desktop**
image
- **Mobile**
image

### Reviewers

- [x] @catarinapm 
- [x] @mariasbl

### CODE REVIEW

File: fe/i18n/messages/en.json [hunk 1]
```diff
@@ -30,6 +30,10 @@
     },
     "EventInfo": {
       "level": "Level:"
+    },
+    "Stepper": {
+      "textOne": "PAYMENT INFO",
```
Comment (mariasbl) [inline fe/i18n/messages/en.json:35]: Can you change the text to camelcase instead of capslock?

**Review by mariasbl — CHANGES_REQUESTED** [review]

File: fe/i18n/messages/en.json (hunk 1 above)
Comment (danielaprado) [reply fe/i18n/messages/en.json:35]: But to be on the screen with Camelcase?

Comment (mariasbl) [reply fe/i18n/messages/en.json:35]: No, but i think it makes more sense for the translations to be camelcase and for the component to be uppercase for any text that comes in.

**Review by catarinapm — APPROVED** [review]

**Comment by catarinapm** [comment]
I suggest that in the future we add the possibility to click on filled steps - to change some info if needed

**Review by mariasbl — APPROVED** [review]

**Pull request merged**